packages/api/helper/mailer.js

```diff
@@ -37,16 +40,34 @@ function sendDeclineEmail(nomination) {
   email.send({
     template: 'decline',
     message: {
-      from: 'Bill <bill@keepswimmingfoundation.org>',
-      // to: 'sophia@thedifferenceengine.co'
+      from: adminEmail,
       to: nomination.providerEmailAddress,
     },
     locals: {
       name: nomination.providerName,
       patientName: nomination.patientName,
       appUrl: process.env.APP_URL,
     }
-  }.catch((err) => console.log(err))).then(() => console.log('email has been sent!'));
+  }).catch((err) => console.log(err)).then(() => console.log('email has been sent!'));
+}
+
+function sendSurveyEmail(nomination) {
+  email.send({
+    template: 'survey',
+    attachments: './survey/header.jpg',
+    message: {
+      from: adminEmail,
+      to: nomination.providerEmailAddress
+      // to: 'mark@thedifferenceengine.io'
```

Comment:
We can probably remove this commented line

Reply:
I will remove it.